Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:6 AS builder
+
+ENV ENV_NODE=production
+
+RUN groupadd --gid 1100 lisk && \
+    useradd --create-home --home-dir /home/lisk --shell /bin/bash --uid 1100 --gid 1100 lisk
+# As of May 2018 cloud.docker.com runs docker 17.06.1-ce
+# however version 17.12 is required to use the chown flag
+COPY . /home/lisk/lisk/
+RUN chown lisk:lisk --recursive /home/lisk/lisk
+
+USER lisk
+WORKDIR /home/lisk/lisk
+
+RUN npm install
+
+
+FROM node:6
```

Comment:
Duplicate line.

Reply:
This is a different container (the final one, not the build container)